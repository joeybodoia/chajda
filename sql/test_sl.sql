CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('sl',' "To je moj primer testnega stavka');
