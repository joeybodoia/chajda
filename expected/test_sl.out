CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
    spacy_lemmatize    
-----------------------
   testnega:6 stavka:7
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
  spacy_lemmatize  
-------------------
   testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                spacy_lemmatize                 
------------------------------------------------
   to:2 je:3 moj:4 primer:5 testnega:6 stavka:7
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
          spacy_lemmatize           
------------------------------------
   to je moj primer testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
      spacy_lemmatize      
---------------------------
   ":2 testnega:7 stavka:8
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
   spacy_lemmatize   
---------------------
   " testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                  spacy_lemmatize                   
----------------------------------------------------
   ":2 to:3 je:4 moj:5 primer:6 testnega:7 stavka:8
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
           spacy_lemmatize            
--------------------------------------
   " to je moj primer testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
    spacy_lemmatize    
-----------------------
   testnega:6 stavka:7
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
  spacy_lemmatize  
-------------------
   testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                spacy_lemmatize                 
------------------------------------------------
   To:2 je:3 moj:4 primer:5 testnega:6 stavka:7
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
          spacy_lemmatize           
------------------------------------
   To je moj primer testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
      spacy_lemmatize      
---------------------------
   ":2 testnega:7 stavka:8
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
   spacy_lemmatize   
---------------------
   " testnega stavka
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                  spacy_lemmatize                   
----------------------------------------------------
   ":2 To:3 je:4 moj:5 primer:6 testnega:7 stavka:8
(1 row)

SELECT spacy_lemmatize('sl',' "To je moj primer testnega stavka' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
           spacy_lemmatize            
--------------------------------------
   " To je moj primer testnega stavka
(1 row)

SELECT spacy_tsvector('sl',' "To je moj primer testnega stavka');
     spacy_tsvector      
-------------------------
 'stavka':7 'testnega':6
(1 row)

